Cargo.toml

```diff
@@ -73,25 +73,21 @@ ripemd = "0.1"
 secp256k1 = "0.30"
 sha-1 = "0.10"
 sha2 = "0.10"
-tracing = "0.1.41"
```

Comment:
Bumping this down to not add tight version restrictions to crate users. I tested that 0.1.0 works by using `cargo +nightly update -Z direct-minimal-versions` (we can add that test to CI later)